src/main/java/com/kmush12/CurrencyConverter/Converter/CurrencyConverterService.java

```diff
@@ -1,30 +1,33 @@
 package com.kmush12.CurrencyConverter.Converter;
 
-
 import com.kmush12.CurrencyConverter.Exchange.ExchangeRate;
 import com.kmush12.CurrencyConverter.Exchange.ExchangeRequest;
 import com.kmush12.CurrencyConverter.Exchange.ExchangeResult;
+import com.kmush12.CurrencyConverter.urlReader.PageReader;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
+import java.io.IOException;
 import java.math.BigDecimal;
-import java.util.Arrays;
-import java.util.List;
 import java.util.Optional;
 
-
 @Service
 public class CurrencyConverterService {
 
-    private final List<ExchangeRate> exchangeRatesList = Arrays.asList(new ExchangeRate(0, "EUR", "USD", new BigDecimal("0.96")),
-            new ExchangeRate(1, "USD", "EUR", new BigDecimal("1.05")),
-            new ExchangeRate(2, "EUR", "PLN", new BigDecimal("4.7")),
-            new ExchangeRate(3, "PLN", "EUR", new BigDecimal("0.21")),
-            new ExchangeRate(4, "USD", "PLN", new BigDecimal("4.6")),
-            new ExchangeRate(5, "PLN", "USD", new BigDecimal("0.2"))
-    );
+    @Value("${getdata.url}")
+    private String link;
+
+    public String createCurrencyList() throws IOException {
+        String pathToCurrencyList = link + "symbols";
+        PageReader reader = new PageReader();
+        JSONObject all = reader.getPageContent(pathToCurrencyList);
+        return all.getJSONObject("symbols").toString();
```

Comment:
To ogólnie działa, ale to jest taki standard pobierania danych z innych serwisów z roku max 2000. Używasz starego API z jakichś pierwszych wersji Javy.

Zobacz jak tworzyć klientów zewnętrzych serwisów RESTowych. Popatrz na takie frameworki jak Feign, RestTemplate, OkHttpClient, czy inne. Podpowiem, że najprościej i najczyściej implementacyjnie będzie jak użyjesz Feigna.

Poza tym cała ta logika powinna być zamknięta w jakiejś innej klasie, np `CurrencyRateClient` czy coś takiego.